pyproject.toml

```diff
@@ -1,18 +1,22 @@
 [tool.poetry]
 name = "canonicalwebteam.flask_base"
-version = "0.2.0"
+version = "0.3.0"
 description = ""
 authors = ["Canonical webteam <webteam@canonical.com>"]
 packages = [{ include = "canonicalwebteam" }]
+readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3"
+python = "^3.6"
 "canonicalwebteam.yaml-responses" = {version = "^1.1", extras = ["flask"]}
 flask = "1.0.2"
-talisker = {version = "0.14.3", extras = ["gunicorn"]}
+talisker = {version = "0.14.3", extras = ["gunicorn", "flask", "prometheus", "raven"]}
+poetry-setup = "^0.3.6"
```

Comment:
Does this need to be set in dependencies and dev dependencies?

Reply:
No, I'll remove that